src/Layout/__tests__/__snapshots__/Layout.test.js.snap

```diff
@@ -2,17 +2,206 @@
 
 exports[`Layout should match snapshot 1`] = `
 <div
-  className="jsx-4212418802 Layout"
+  className="Layout"
 >
-  Aloha, I am a 
-  <span
-    className="jsx-4212418802 name"
-  >
-    Layout
-  </span>
+  <link
+    href="https://fonts.googleapis.com/css?family=Roboto"
+    rel="stylesheet"
+  />
+  <Intro />
   <JSXStyle
-    css="*.jsx-4212418802{font-family:Roboto;}div.Layout.jsx-4212418802 span.name.jsx-4212418802{color:blue;}"
-    styleId="4212418802"
+    css={
+      String {
```

Comment:
It is true that is weird. I figured they appear differently because these styles are the ones injected as `global` not as component-level styles, so surely must be compiled in some other different way? https://github.com/kiwicom/smart-faq/pull/18/files/8a8d6e8f2f4542a055bae00fe74c14587696fc44#diff-7debb7cc791b5268e64f8e2ff2cb49caR31

I'll have a look at it tho.